lib/kitchen/directions/bake_chapter_content_module.rb

```diff
@@ -0,0 +1,13 @@
+module Kitchen
+  module Directions
+    module BakeChapterContentModule
+      def self.v1(book:)
+        book.chapters.each do |chapter|
+          chapter.search('[data-type="abstract"]').each do |abstract|
+            abstract.prepend(child:'<h3 data-type="title">Learning Objectives</h3>')
```

Comment:
Use an I18n translation here instead of hardcoding the text.  Make sure to stub the translation in the spec